src/lgdo/lh5/core.py

```diff
@@ -218,13 +277,28 @@ def write(
     write_start
         row in the output file (if already existing) to start overwriting
         from.
+    page_buffer
+        enable paged aggregation with a buffer of this size in bytes
+        Only used when creating a new file. Useful when writing a file
+        with a large number of small datasets. This is a short-hand for
+        (fs_stragety="page", fs_pagesize=[page_buffer], fs_persist=True,
+        fs_threshold=1)
     **h5py_kwargs
         additional keyword arguments forwarded to
         :meth:`h5py.Group.create_dataset` to specify, for example, an HDF5
         compression filter to be applied before writing non-scalar
         datasets. **Note: `compression` Ignored if compression is specified
         as an `obj` attribute.**
     """
+    if wo_mode in ("w", "write", "of", "overwrite_file"):
+        h5py_kwargs.update(
+            {
+                "fs_strategy": "page",
+                "fs_page_size": page_buffer,
+                "fs_persist": True,
+                "fs_threshold": 1,
```

Comment:
If these values are already the default ones in h5py, why explicitly setting them here? are they needed for the paged buffer functionality to work?

Reply:
I don't know why I made these explicit, but I don't think it needs to be.